crates/pallet-balanced-currency-swap-bridges-initializer/src/lib.rs

```diff
@@ -96,6 +93,13 @@ pub mod pallet {
         /// The evm-native bridge pot account.
         type EvmNativeBridgePot: Get<Self::EvmAccountId>;
 
+        /// The current bridges initializer version.
+        type InitializerVersion: Get<u16>;
```

Comment:
To avoid the confusion with the `CURRENT_BRIDGES_INITIALIZER_VERSION` and the version of the pallet state semantics we should call this something else.

Reply:
Redesigned based on discussion.